doc/source/installation.rst

```diff
@@ -157,22 +232,21 @@ For example, ``extra_cflags`` can be used to tune your MDAnalysis installation f
 
 Use of these flags can give a significant performance boost where the compiler can effectively autovectorise.
 
-Be sure to use the recommended flags for your target architecture. For example, ARM platforms recommend using ``-mcpu`` *instead* of ``-mcpu``, while
-PowerPC platforms prefer *both* ``-mcpu`` and ``-mtune``.
+Be sure to use the recommended flags for your target architecture. For example, ARM platforms recommend using ``-mcpu`` instead of ``-march``, while 
```

Comment:
Is this still the case? GCC documentation now says the following about `-march=name`:

> This option can be used in conjunction with or instead of the -mcpu= option.

And under `-mcpu`:

> Where this option is used in conjunction with `-march` or `-mtune`, those options take precedence over the appropriate part of this option.

I'd double check if this information is still relevant. I'd point directly to the compiler documentation here, because what we write here might get out of date at some point.